medusa/common.py

```diff
@@ -507,16 +508,16 @@ def scene_quality_from_name(name, quality):
             elif 'uhdtv' in name:
                 rel_type = ' UHDTV'
 
-        # If SDDVD
-        if quality == 2:
+        if quality == Quality.SDDVD:
             rel_type = ' BDRip'
             if re.search(r'br(-| |\.)?(rip|mux)', name):
                 rel_type = ' BRRip'
             elif re.search(r'dvd(-| |\.)?(rip|mux)', name):
                 rel_type = ' DVDRip'
 
         # If any WEB type
-        if quality in (32, 64, 1024, 8192):
+        if quality in (Quality.HDWEBDL, Quality.FULLHDWEBDL, Quality.UHD_4K_WEBDL,
```

Comment:
Shouldnt we create groups for these?

Reply:
I don't see it as necessary.

Reply:
My opinion is we should, **but only if** it's needed in more than one or two places.